modules/ROOT/pages/anypoint-insights.adoc

```diff
@@ -0,0 +1,504 @@
++
+Get up to the last 30 days of metrics on latency, requests, responses, errors, Mule messages and the JVM for Mule apps, and for API implementations, data on policy violations, performance, and client apps. {download-chart-data}
+
+[[entities]]
+include::partial$include-entities.adoc[tags="entity-definition"]
+
+[[dashboard-overview]]
+//don't change order of numbers in graphic or calloutlist
+image::insights.png["Anypoint Insights UI"]
+
+[calloutlist]
+. Dashboard that provides an overview of metrics across of your deployed Mule apps and APIs
+. Environment selector for your deployments (such as Sandbox or Production)
+. <<filter, Filters>> to narrow the data in charts
+. Performance <<charts-performance, metrics and trends>> across your deployed Mule apps and APIs
+. <<entities-list, List of each Mule app and API>> that you can click to display <<charts-detailed, detailed metrics>> on a specific deployment
+. Selector for the time range covered by the charts 
++
+{selector-all-dashboards}
+. Selector for auto-refresh interval, displayed when auto-refresh is enabled 
++
+{selector-all-dashboards}
+
+[[prereqs]]
+== Before You Begin
+
+Required prerequisites:
+
+* An Anypoint Platform account in a control plane that supports Anypoint Insights:
+
+** {empty}
+include::partial$include-entities.adoc[tags="anypoint-regions-hyperforce"] 
+
+** {empty}
+include::partial$include-entities.adoc[tags="anypoint-regions-useu"]
+
+* A Mule app or API (xref:gateway-home::index.adoc[Mule Gateway or Flex Gateway API]) running on a supported version of Mule Runtime engine (see xref:release-notes::mule-runtime/mule-esb.adoc[]) and deployed to a supported target:
+
+** CloudHub 2.0
+//TEMP INFO: Hyperforce 12/24 (W-17225983)
+** CloudHub (for US and EU clouds, in Advanced and Titanium tiers only)
+//TEMP INFO: Hyperforce 12/24 (W-17225983)
+** Hybrid (for US and EU clouds, in Advanced and Titanium tiers only)
+** Runtime Fabric
+
+* For Canada Cloud and Japan Cloud: 
+
+** Enable Mule runtime access to Anypoint Platform so that you can display entity metrics in Anypoint Monitoring.
++
+For guidance, see xref:hyperforce::configure-mule-hyperforce.adoc[].
+** For information about Runtime Fabric feature support in these clouds, see xref:hyperforce::index.adoc#features-supported-on-hyperforce[Features Supported on Hyperforce].
+
+* The *Monitoring Viewer* permission to Anypoint Monitoring and the permissions required for viewing apps and APIs (see xref:monitoring::am-permissions.adoc[])
+
+[[monitor-deployments]]
+== Monitor Your Deployments
+
+Monitor Mule apps and APIs within a selected business group and environment.
+
+. Meet the prerequisites in <<prereqs>>. 
+. Log in to Anypoint Platform, and navigate to Anypoint Monitoring. 
+//Nav step
+include::partial$include-entities.adoc[tags="open-insights"]
+. In Anypoint Insights, set your environment, such as Design, Production, and Sandbox. 
++
+For guidance, see <<change-environment>>.
```

Comment:
For these "For guidance", you could consider using inline links:
"<<change-environment,set your environment>>"

Reply:
@dukesphere [W-17361464](https://gus.lightning.force.com/a07EE000026JsovYAC) future

Reply:
The more I think about this, I'd rather people not click to get how-to info on a step unless they decide they need it. I think they're more likely to click the link to the info if that link is part of the step vs. in a sentence below.